.gitignore

```diff
@@ -67,3 +67,11 @@ docs/external_examples.rst
 
 # vim
 *.swp
+
+<<<<<<< Updated upstream
+# venv
+.venv
+=======
+# virtual environments
+.venv/
+>>>>>>> Stashed changes
```

Comment:
Conflict seems not solved.

Reply:
Ya, nearly all the tests are failing for me in this repo and several simply do not run:

![image](https://user-images.githubusercontent.com/59346180/175796946-34276270-806b-4e11-85f5-426e021667c7.png)

Reply:
I think this will need some TLC ❤️